documentation/advanced/tutorial-i18n-localization.asciidoc

```diff
@@ -133,7 +133,7 @@ public class TranslationProvider implements I18NProvider {
 }
 ----
 
-=== Using localization in the application
+=== Using Localization in the Application
 
 Using the internationalization in the application is a combination of using the
```

Comment:
A bit of a mismatch here with “using localization” in the heading and then “using the internationalization” in the text. Minor detail, just wondering if anyone knows how to properly use these terms (I don’t)? 😅

Reply:
Me neither 🤷